doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

```diff
@@ -109,22 +108,13 @@ No application input is required to handle the requests.
 Bluetooth authentication
 ========================
 
-The Fast Pair service overrides the set of Zephyr's Bluetooth authentication callbacks (:c:struct:`bt_conn_auth_cb`) while handling Key-Based Pairing request.
-The callbacks are updated to report proper device capabilities for Bluetooth LE pairing request/response packets.
-Overriding callbacks allows the GFPS to take over Bluetooth authentication during `Fast Pair Procedure`_ and perform all of the required operations without interacting with the application.
-After the procedure is finished, the previously used callbacks are assigned back.
+The Bluetooth pairing is handled using a set of Bluetooth authentication callbacks (:c:struct:`bt_conn_auth_cb`).
+The pairing flow and the set of Bluetooth authentication callbacks in use depend on the connected peer:
 
-The Fast Pair Provider can still bond using a normal Bluetooth LE bonding procedures, without using Fast Pair.
-The normal Bluetooth LE bonding procedure can be used, for example, if the connected Bluetooth Central does not support Fast Pair.
-In that case, the bonding uses Bluetooth authentication callbacks registered by the application.
-
-.. note::
-   Zephyr supports only one set of Bluetooth authentication callbacks.
-   Because of this limitation, it is not possible to simultaneously handle bonding using Fast Pair procedure and normal Bluetooth LE bonding.
-   In that case improper device capabilities would be reported.
-
-   Make sure that the normal Bluetooth LE bonding procedure and Fast Pair procedure do no overlap with each other.
-   The recommended approach is to avoid handling more than one bonding procedure at a time.
+* If the connected peer follows the Fast Pair pairing flow, the Fast Pair service calls :c:func:`bt_conn_auth_cb_overlay` to automatically overlay the Bluetooth authentication callbacks.
+  The function is called while handling the Key-Based Pairing request.
+  Overlying callbacks allows the GFPS to take over Bluetooth authentication during `Fast Pair Procedure`_ and perform all of the required operations without interacting with the application.
+* Otherwise, if the connected peer does not follow the Fast Pair pairing flow, normal Bluetooth LE pairing and global Bluetooth authentication callbacks are used.
```

Comment:
I think we could remove the bullets (*) and have a paragraph break after line 116.

Reply:
Changed

Reply:
I think a compromise could be having the following introductory sentence:
```
The pairing flow and the set of Bluetooth authentication callbacks in use depend on the connected peer and whether it follows the Fast Pair pairing flow:
```
And then the bullets introduced in similar fashion:
```
* If it follows the pairing flow, [...]
* If it does not follow the pairing flow, [...]
```
Personally, I always prefer bullets in similar situations, as they clearly show that some things are related to each other, while paragraphs might be about different topics. BUT two separate paragraphs would still be acceptable here. So it all depends on the phrasing and context.